shipyard_blueprints/shipyard-bp-utils/shipyard_bp_utils/files.py

```diff
@@ -231,12 +232,33 @@ def find_all_local_file_names(source_folder_name: str = None) -> list:
     list: A list of all files that exist in the current working directory.
     """
     logger.debug(f"Finding all local file names in {source_folder_name}...")
-    cwd = os.getcwd()
-    cwd_extension = os.path.normpath(f"{cwd}/{source_folder_name}/**")
-    all_paths = glob.glob(cwd_extension, recursive=True)
-    logger.debug(f"Found {len(all_paths)}.")
-    logger.debug(all_paths)
-    return remove_directories_from_path_list(all_paths)
+    # cwd = os.getcwd()
```

Comment:
I will remove this commented out code upon approval, just wanted to get your thoughts on the change